### PR TITLE
Null handling in divide

### PR DESCRIPTION
**Pull request by clinique**
This is my proposal to address what I think is an issue #488

### CODE REVIEW

File: bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/AbstractMathTransformationService.java [hunk 1]
```diff
@@ -34,27 +34,29 @@ abstract class AbstractMathTransformationService implements TransformationServic
...
-            throw new TransformationException("Math Transformation can only be used with numeric inputs");
-        }
-        QuantityType<?> value;
-        try {
-            value = new QuantityType<>(valueString);
-        } catch (IllegalArgumentException e) {
-            logger.warn("Input value '{}' could not be converted to a valid number", valueString);
-            throw new TransformationException("Math Transformation can only be used with numeric inputs");
+        
+        if("NULL".equals(valueString) || "NULL".equals(sourceString)) {
```
Comment (J-N-K) [inline bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/AbstractMathTransformationService.java:38]: Wouldn't it make sense to check `UNDEF`, too?

Comment (J-N-K) [reply bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/AbstractMathTransformationService.java:38]: @clinique Can you comment here? I would like to do a release in the next days due to #490.

Comment (clinique) [reply bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/AbstractMathTransformationService.java:38]: > Wouldn't it make sense to check `UNDEF`, too?

You're correct, I thought of it but I only faced the error with NULL, never UNDEF, so thought : if not broken, don't fix it :)

**Comment by J-N-K** [comment]
I added another version to get it into 3.2.17

**Pull request closed without merging**